src/main/java/com/softserve/kickscooter/kickscootervehicle/management/controller/ManagementController.java

```diff
@@ -0,0 +1,66 @@
+package com.softserve.kickscooter.kickscootervehicle.management.controller;
+
+import com.softserve.kickscooter.kickscootervehicle.management.dao.Scooter;
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterCreateDto;
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterInfoDto;
+import com.softserve.kickscooter.kickscootervehicle.management.service.ManagementService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Slf4j
+@RestController
+@RequestMapping("/scooter")
+@AllArgsConstructor
+public class ManagementController {
+
+    private ManagementService service;
+
+    @GetMapping("/id")
+    public ResponseEntity<Iterable<UUID>> getScootersId(){
+        log.debug("getScootersId method");
+        log.debug(service.toString());
+        return ResponseEntity.ok(service.getScootersId());
+    }
+
+    @GetMapping("/info-all")
```

Comment:
Does the purpose is to get all scooters? If so, remove `("/info-all")`, so the line will container just `@GetMapping` annotation